backup-loop.sh

```diff
@@ -75,17 +79,16 @@ while true; do
       fi
 
     fi
-
     rcon-cli save-on
   else
     log ERROR "rcon save-off command failed"
   fi
 
-  if (( PRUNE_BACKUPS_DAYS > 0 )); then
+  if (( PRUNE_BACKUPS_DAYS > 0 )) && [ -n "$(find_old_backups -print -quit)" ]; then
     log INFO "pruning backup files older than ${PRUNE_BACKUPS_DAYS} days"
-    find "${DEST_DIR}" -mtime "+${PRUNE_BACKUPS_DAYS}" -delete
+    find_old_backups -print -delete
```

Comment:
Why was this change needed?

Reply:
What do you mean?

Extracting it to separate function? So that I can reuse the same code for checking if there's any backups to delete - so that it won't log that it's deleting something while it's not doing that.

Adding -print ? I thought it might be useful to list deleted backups. Might want to add timestamps to it though, with log function.

Let me know if you don't think that any of these changes are desired.

Reply:
Thanks, you clarified it for me. Basically I was just wondering what the overall difference was other than adding `-print`, but now I see there's more in store for the refactoring.